src/web/routes/application/select-address/select-address.js

```diff
@@ -20,6 +20,8 @@ const buildAddressOption = result => ({
 })
 
 const behaviourForGet = () => (req, res, next) => {
+  // reset this value as this is used to determine whether or not to skip the manual address page
+  delete req.session.claim.selectedAddress
```

Comment:
I'm not sure I understand why this is needed, lets discuss when you get in.

Reply:
Might be worth abstracting this to a named function to clarify the purpose. Something like:
```
const resetAddressState = (req) => {
  if (req.session.claim.selectedAddress) {
    delete req.session.claim.selectedAddress
  }
}
```

Reply:
Which could also be unit tested

Reply:
Moved into own function and updated behaviourForGet unit test.